docs/2021/project-planning.md

```diff
@@ -0,0 +1,68 @@
+# Project Planning
+
+### Week 0 (08 March - 14 March)
+* Analyze GitHub API
+
+### Week 1 (15 March - 21 March)
+* Analyze GitHub Classroom
+* Sketch the UI page/representation interaction
+* Create and Deliver the project form
+
+### Week 2 (22 March - 28 March)
+* Research [JDBI's](https://jdbi.org/) fluent API
+* Design the data model
+
+### Week 3 (05 April - 11 April)
+* Define authentication and access token architecture
+* Create and Deliver the project proposal
+
+### Week 4 (12 April - 18 April)
+* Tweak the physical data model to support the new authentication architecture
+* Start developing the API (using [Spring](https://spring.io/))
+
+### Week 5 (19 April - 25 April)
+* Further develop the API
+* Start developing API tests
+* Start documenting the API
+
+### Week 6 (26 April - 02 May)
+* Start developing a proof of concept (POC) for Web App
+* Research hosting possibilities for CodeGarten
+
+### Week 7 (03 May - 09 May)
+* Final touches in order to have a working POC of CodeGarten
+* Deploy the beta version of CodeGarten
+
+### Week 8 (10 May - 16 May)
+* Implement continuous integration for automatic testing of the API

```

Comment:
CI should be placed before, event before we have tests (in that moment CI just ensures the build succeeds).